### ISSUE TITLE
Another glitch in addon manager compact view

### ISSUE THREAD

**Issue by Speravir** [Aris-t2/CustomCSSforFx#424]
Followup to #418:

I now observe with 4.1.1 almost the same shift of the panel list, but in the expanded view (detailed view for an addon):
![Add-ons-Verwaltung_1](https://user-images.githubusercontent.com/899244/140843718-600681bd-a9dc-4fa0-87c0-dcf9cecf7ac2.png)

I was able to fix it with
```css
addon-card[expanded="true"] panel-list {
	margin-top: -30px !important;
}
```
but this may only look good on my computer depending on font-size settings:
![Add-ons-Verwaltung_2](https://user-images.githubusercontent.com/899244/140844803-fe908660-460a-46f4-8b85-396381df51b2.png)

 I noticed in devtools view that there is a hidden  `.addon-description` (without research `display:none` I guess), so this could be the actual issue here like in the other issue.

**Comment by Aris-t2** [comment]
I was trying to fix some other add-on manager issues and changed a few things already. Could you run the test build I posted [here](https://github.com/Aris-t2/CustomCSSforFx/issues/423) a few days ago?

https://github.com/Aris-t2/CustomCSSforFx/issues/423

**Comment by Speravir** [comment]
Yes, it works - kind of. But I noticed an embarrassing thing: I did _not_ have enabled `addonlists_show_buttons_instead_of_menu_popup_compact_view.css`, but the “uncompact” version, and now I am not certain whether the issue came from this. I had not observed problems regarding this earlier, though, otherwise I’d had realized this earlier.

For the “working - kind of”: Look at this (btw I could narrow the window more than yesterday, I guess due to your changes):
![Add-ons-Verwaltung_3](https://user-images.githubusercontent.com/899244/141019822-d4ce51a0-4b38-425b-a1d1-474f3b867f6c.png)
A slight bit (more?) top margin or padding for `addon-card:not([expanded="true"])` would obviously be good. But I am unsure here, too, whether this is only on my side.

As a side note: You do not see the search form, because I’ve hidden it (with `visibility: hidden`, because `display: none` did more than I wanted).
